proto/scheduler.proto

```diff
@@ -203,6 +203,11 @@ message ExecutionNode {
   //
   // Ex. "34c5cf7e-b3b1-4e20-b43c-3e196b30d983"
   string executor_id = 9;
+
+  // Name of the execution server this executor is running on
+  //
+  // Ex. "8BiY6U0F"
```

Comment:
I think it would look nicer if we displayed this value as a UUID rather than the base64 encoded value of the UUID. 
base64 encoding can have characters like + / etc which doesn't look very nice. 

@tylerwilliams Do you have any opinions on this?

Reply:
No preference, as long as we can make it clickable in the UI. I don't think these are full UUIDs though, they are just the host part of the ID, right?

(We have already written many of these in the currrent format to the executions table so there's some inertia there...)

Reply:
Yep, these are the node IDs used by UUID (which is to say, generally, the MAC addresses) encoded to string by the base64 library. If we want to change how we're formatting the ID, given that it generally is a MAC address, maybe we should format it like one?

Reply:
After talking with Vadim and Tyler offline, it looks like the direction we're going with this is to just use GUIDs (formatted normally) that are stored to disk, since MAC addresses can get re-used. I'll address this in a separate PR, since it's a bit beyond the scope of this one.

Reply:
(gonna wait on reviewing this CL till after that one is in)

Reply:
https://github.com/buildbuddy-io/buildbuddy/pull/1400